src/software/ai/navigator/path_planner/enlsvg_path_planner_test.cpp

```diff
@@ -280,11 +278,11 @@ TEST_F(TestEnlsvgPathPlanner, test_enlsvg_path_planner_same_cell_dest)
 
     EXPECT_TRUE(path != std::nullopt);
 
-    std::vector<Point> path_points = path->getKnots();
+    std::vector<Point> path_points = path.value();
```

Comment:
we shouldn't need this variable since we don't use it, same with similar examples in this pr